core/src/main/java/org/apache/accumulo/core/clientImpl/ClientContext.java

```diff
@@ -260,18 +261,20 @@ public Ample getAmple() {
       submitScannerReadAheadTask(Callable<List<KeyValue>> c) {
     ensureOpen();
     if (scannerReadaheadPool == null) {
-      scannerReadaheadPool = clientThreadPools.getPoolBuilder("Accumulo scanner read ahead thread")
+      scannerReadaheadPool = clientThreadPools
+          .getPoolBuilder(METRICS_POOL_PREFIX + "client.context.scanner.read.ahead")
```

Comment:
I'm not sure that we should put the location into the metric names. If the location changes between versions then that could break users monitoring.

Reply:
I thought Keith suggested adding client - what do you feel is a better name?

Reply:
IMO, the metric names should reflect the function without the location. The location of the metric is emitted as the process name tag. For the client, none of these metrics will be emitted unless the user wires up Micrometer in their client.

Reply:
In this case, the pool does not emit metrics.